.gitignore

```diff
@@ -1,4 +1,7 @@
 
+# pepr-test-module should be ignored
+pepr-test-module/
```

Comment:
this is already on line 21, was there a reason to add it here too?

Reply:
oversight, saw it in my editor as new code and didn't realize